packages/debugger-extension/src/index.ts

```diff
@@ -478,6 +478,19 @@ const variables: JupyterFrontEndPlugin<void> = {
         });
       }
     });
+
+    commands.addCommand(CommandIDs.copyToClipboard, {
+      label: 'Copy to Clipboard',
+      caption: trans.__('Copy value to clipboard'),
```

Comment:
Maybe:

```suggestion
      caption: trans.__('Copy text representation to clipboard'),
```

Not sure if that would be correct.

Reply:
I changed it to 'Copy text representation of the value to clipboard'